### PR TITLE
Scraping union

### PR DESCRIPTION
**Pull request by YehuditShrg**
testing the screpers

### CODE REVIEW

File: Scrapers/CompanyScrapers/InnodataScraper.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            title, location = ul_tag.findNext('span', {'class': 'elementor-icon-list-text'}).text.strip().split('-')
+            # can not get specifically to the modal although i took the modal id but it brings to the specific place
+            # of the job on the web page
+            link = self.url
+            self.positions.append(self.Position(
+                title=title,
+                link=link,
+                location=location.strip()
+            ))
+InnodataScraper().check_self()
```
Comment (snirsh) [inline Scrapers/CompanyScrapers/InnodataScraper.py:43]: This is probably the Error.

**Review by snirsh — CHANGES_REQUESTED** [review]

File: Scrapers/CompanyScrapers/InnodataScraper.py (hunk 1 above)
Comment (snirsh) [inline Scrapers/CompanyScrapers/InnodataScraper.py:43]:
```suggestion

```

File: Scrapers/CompanyScrapers/MdihealthScraper.py
Comment (snirsh) [inline Scrapers/CompanyScrapers/MdihealthScraper.py:1]: Empty file, remove

Comment (snirsh) [reply Scrapers/CompanyScrapers/MdihealthScraper.py:1]: I removed it myself

**Comment by YehuditShrg** [comment]
the branch is ready to merge with the main!!

**Review by snirsh — DISMISSED** [review]
LGTM

**Comment by snirsh** [comment]
LGTM

File: Scrapers/CompanyScrapers/SquareTalkScraper.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from Scrapers.Scraper import *
+from selenium.webdriver.common.by import By
+
+
+class SquareTalkScraper(Scraper):
+    name = 'SquareTalk'
+    url = 'https://squaretalk.com/career/#rec_job_listing_div'
+
+    def scrape(self):
+        driver = self.selenium_url_maker(self.url)
```
Comment (snirsh) [inline Scrapers/CompanyScrapers/SquareTalkScraper.py:10]: Can't work either soup only? Without selenium?

**Review by snirsh — CHANGES_REQUESTED** [review]

File: Scrapers/CompanyScrapers/SquareTalkScraper.py (hunk 2 above)
Comment (YehuditShrg) [reply Scrapers/CompanyScrapers/SquareTalkScraper.py]: I removed this scraper to a new branch,
I'll try to do it tomorrow אי"ה.
(at first glance it seems that you must wait for the elements in this page)

**Review by snirsh — APPROVED** [review]

**Pull request merged**